### PR TITLE
gradle: update to 7.3

### PR DESCRIPTION
**Pull request by masterjavaofficial**

### CODE REVIEW

File: packages/gradle/build.sh [hunk 1]
```diff
@@ -2,10 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://gradle.org/
 TERMUX_PKG_DESCRIPTION="Powerful build system for the JVM"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=1:7.2
+TERMUX_PKG_VERSION=7.3
```
Comment (thunder-coding) [inline packages/gradle/build.sh:5]: Why are you removing package epoch? It isn't supposed to be removed even while updating the package

**Review by thunder-coding — CHANGES_REQUESTED** [review]

File: packages/gradle/build.sh (hunk 1 above)
Comment (masterjavaofficial) [reply packages/gradle/build.sh]: Changes committed. Now its ok?

**Review by thunder-coding — APPROVED** [review]
lgtm

File: packages/gradle/build.sh [hunk 2]
```diff
@@ -2,10 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://gradle.org/
 TERMUX_PKG_DESCRIPTION="Powerful build system for the JVM"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=1:7.2
+TERMUX_PKG_VERSION=1:7.3
 TERMUX_PKG_REVISION=1
```
[the remark this answers is not in the document]
Comment (thunder-coding) [reply packages/gradle/build.sh:6]: Oh yeah, how did I miss that 😅

**Pull request merged**